CITATION.cff

```diff
@@ -25,8 +25,8 @@ identifiers:
   - type: doi
     value: 10.5281/zenodo.7696031
     description: Latest archived version of Metasynth
-repository-code: 'https://github.com/sodascience/metasynth'
-url: 'https://metasynth.readthedocs.io/'
+repository-code: 'https://github.com/sodascience/metasyn'
+url: 'https://metasyn.readthedocs.io/'
```

Comment:
This link is currently broken - do we have to manually make a rtd page for metasyn? or will the current page automatically be renamed?

Reply:
Yes, I think @vankesteren needs to do that manually.